src/payment/strategies/zip/zip-payment-strategy.ts

```diff
@@ -1,3 +1,6 @@
+import { FormPoster } from '@bigcommerce/form-poster';
+import { RequestSender, Response } from '@bigcommerce/request-sender';
+
```

Comment:
Why this extra break line?

Reply:
It's to separate the imports that are in `node_modules` from the rest of imports.